### ISSUE TITLE
Libraries with json schema types do not render

### ISSUE THREAD

**Issue by tfayruzov** [mulesoft/api-console#388]
I try to modularize my RAML project and create a library. When a type in library is defined as RAML console works, but when it is references as JSON schema console fails:

Sample schema:
```
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "id": "<URL that is actually published>",
  "type": "object",
  "properties": {
    "language": {"type": "string", "minLength": 2},
    "country": {"type": "string", "minLength": 2}
  },
  "required": ["language"]
}
```
Library:
```
#%RAML 1.0 Library
types:
  Locale:
    type: !include schemas/locale.schema
```
API:
```
#%RAML 1.0
title: Library test
uses:
  primitives: library.raml
/exampleRes:
  post:
    body:
      application/json:
	type: primitives.Locale
```

Getting error: `[error] Inheriting from unknown type`

**Comment by nmarinel01** [comment]
Hi @tfayruzov ! Thanks for reporting this.
This might be related to the fact that libraries references are not properly handled. We need to check if the new parser version will fix the issue.

**Comment by InfopactMLoos** [comment]
I am experiencing the same issue, could have sworn this worked before.

**Comment by agustinlg** [comment]
Fixed in version 3.0.14.
Thanks
